doc/user_guide/utilities.rst

```diff
@@ -96,3 +96,36 @@ wavelength and pixel position in the detector plane) follows the conventions
 described in the tutorial from  `Horiba Scientific
 <https://horiba.com/uk/scientific/products/optics-tutorial/wavelength-pixel-position>`_.
 
+
+
+Cropping multiple signals in the navigation axis 
+================================================
+
+The function :func:`~.utils.signals.crop_edges`
+removes the specified number of pixels or % from the four edges of a spectral map,
```

Comment:
It looks to me that `crop_units=%` was dropped in favor of HyperSpy's fancy indexing. So the sentence should be rephrased and an appropriate link to the fancy indexing documentation given.